content/docs/start/data-pipelines.md

```diff
@@ -26,11 +26,17 @@ Get the sample code like this:
 $ wget https://code.dvc.org/get-started/code.zip
 $ unzip code.zip
 $ rm -f code.zip
-$ ls src
-evaluate.py  featurization.py  prepare.py
-requirements.txt  train.py
+$ ls -R .
+params.yaml  src/
+
+./src:
+evaluate.py       featurization.py  prepare.py        requirements.txt  train.py
 ```
 
+`params.yaml` contains training and evaluation parameters, we'll cover them later in the guide. 
+
+`src` contains the python code we'll use in this guide. 
+
```

Comment:
Showing params.yaml above seems good to me indeed, since the file is used later in this same page. But these lines explaining what the dir entries are is probably unnecessary, can you roll back (remove) these 4 lines?

Reply:
yeah, src is self-explanatory I think

at this stage we should probably replace `ls` with `tree` to show the better output

`params.yaml` - I would remove "we'll cover them later in the guide. " to avoid creating a feeling that it is something DVC-specific.

Reply:
Agree about `tree` and not explaining src/

I would also try to avoid explaining params.yaml here since it's explained later but if needed, a short sentence as part of the paragraph below should suffice.

Reply:
Yeah, a sentence like "`params` contains params" doesn't really explain much. Let me omit that all together.